README.md

```diff
@@ -97,6 +97,13 @@ const { data: decryptedData, verified } = await decryptMessage({
 **For streamed inputs:**
 to encrypt (and/or sign), pass the stream to `textData` or `binaryData` based on the streamed data type. Similarly, to decrypt and verify, the input options are the same as the non-streaming case. However, if `armoredMessage` (or `binaryMessage`) is a stream, the decryption result needs to be handled differently:
 ```js
+// explicitly loading stream polyfills for legacy browsers is required since v7.2.2
+if (!globalThis.TransformStream || !globalThis.WritableStream) {
+  const ponyfilledStreams = await import('web-streams-polyfill/es6');
+  globalThis.TransformStream = globalThis.TransformStream || ponyfilledStreams.TransformStream;
+  globalThis.WritableStream = globalThis.WritableStream || ponyfilledStreams.WritableStream;
```

Comment:
The way web-steam-tools used to do it is, if one of them is missing, overwrite all of them. And I think that's safer, to avoid unexpected incompatibilities etc. So probably we should do the same here (and in the web apps, if we aren't already):
```suggestion
  ({ ReadableStream, WritableStream, TransformStream } = ponyfilledStreams);
```